### ISSUE TITLE
No tags at all?

### ISSUE THREAD

**Issue by abaxterLHAC** [ApiGen/ApiGen#1097]
Is this project supposed to use tags like @param and @return? I've seen screenshots of them, but they do not show for me and I wasn't sure if I was looking at someone's modification. The only thing from docblocks that is showing is untagged description text. I have tried the two latest alpha versions.

**Comment by JanTvrdik** [comment]
> Is this project supposed to use tags like @param and @return? 

Yes. Can you give me example of code where you used `@param` and `@return` and it did not work?

**Comment by abaxterLHAC** [comment]
Thanks to my own inconsistencies, I actually found one that actually worked and it's because I forgot to put a linebreak on it. It appears that ApiGen does not like the fact that I put the param description on the next line:

The top function works, the bottom doesn't:
![image](https://github.com/ApiGen/ApiGen/assets/46497288/c172f39b-0893-4ed3-9de7-f66545e8a330)
![image](https://github.com/ApiGen/ApiGen/assets/46497288/dfdf9bc1-85a1-437f-b744-f069ce7a9bc4)

I've tried a couple other documentation generators and they both picked up this documentation just fine. 

Also, maybe a different discussion altogether: why doesn't the documentation show @param and @return that include the type and variable name, but omit the description? It would still be useful to see in some cases, for example: PHP 7 and older does not have union type hint and the argument may not be type hinted but the @param would be.

**Comment by JanTvrdik** [comment]
> ApiGen does not like the fact that I put the param description on the next line

Yes, that's how the [phpstan's phpdoc parser](https://github.com/phpstan/phpdoc-parse) (which ApiGen uses) works. It will not pick up the description if it does not start on the same line.

> Why doesn't the documentation show `@param` and `@return` that include the type and variable name, but omit the description?

Can you give me example of code like that?

**Comment by abaxterLHAC** [comment]
I suppose this has become a feature request, then. Starting the description on the next line is convention for [Drupal](https://drupal.org/), possibly more that I am unaware of. What I do not know is if this should be a feature request for ApiGen or for phpstan's phpdoc parser. Having used two other doc generation solutions which support it, it seems to be possible to support both conventions without conflict.

I would also like to point out that next-line descriptions work as expected for @return, but nothing else that I've been able to find: 
![image](https://github.com/ApiGen/ApiGen/assets/46497288/61ba1b06-2d9d-43a8-91b0-c8f242ec78da)
![image](https://github.com/ApiGen/ApiGen/assets/46497288/31b9e373-7ce4-4573-a40b-2553775e5c44)

To answer your question: Please disregard. It initially appeared that the return type was only showing when it was type-hinted, but the return type in the docs appears to use what immediately follows the @return tag. At this time, I do not have any reason to believe that this is not working as expected.
